src/FerriteGmsh.jl

```diff
@@ -187,6 +187,10 @@ Generate a `Ferrite.Grid` from the current active/open model in the Gmsh library
 function togrid(; domain="")
```

Comment:
maybe this should have some `kwarg` as long as the flag functionality is unclear like `overwrite_saveall = true`

and later do `if !saveall_flag && overwrite_saveall`

Reply:
Since this flag is (currently) required I don't see a point of having it configurable. But maybe it is nice to reset it to whatever the original value was before returning from this function?

Reply:
Included it. I added also a test, but I noticed there that the origin point is still in the "grid_bad" its just never used in any element. There should be a better way of doing this, but I cannot figure out why gmsh behaves different